servers/display_server.h

```diff
@@ -558,7 +558,7 @@ class DisplayServer : public Object {
 	virtual int tablet_get_driver_count() const { return 1; };
 	virtual String tablet_get_driver_name(int p_driver) const { return "default"; };
 	virtual String tablet_get_current_driver() const { return "default"; };
-	virtual void tablet_set_current_driver(const String &p_driver){};
+	virtual void tablet_set_current_driver(const String &p_driver) {};
```

Comment:
```suggestion
	virtual void tablet_set_current_driver(const String &p_driver) {}
```

Reply:
I guess I have something with auto formatting outdated. But tests seem to suggest reverting it back like:
```suggestion
	virtual void tablet_set_current_driver(const String &p_driver){};
```
So do you know if/which one is preferred?

Reply:
Yes remove the `;`

Reply:
Great, indeed this one passes too. Thanks!